pkg/collectors/subsystem.go

```diff
@@ -54,9 +54,11 @@ const (
 	SystemPhysicalTotalCapacity = "flashsystem_subsystem_physical_total_capacity_bytes"
 	SystemPhysicalFreeCapacity  = "flashsystem_subsystem_physical_free_capacity_bytes"
 	SystemPhysicalUsedCapacity  = "flashsystem_subsystem_physical_used_capacity_bytes"
+	SystemReclaimableCapacity   = "flashsystem_subsystem_reclamation_capacity_bytes"
```

Comment:
I think we are not exposing this metric, so no need to maintain this label. no?

Reply:
removed